pkg/lib/cockpit-components-table.jsx

```diff
@@ -41,8 +42,11 @@ import './cockpit-components-table.scss';
  * - actions: additional listing-wide actions (displayed next to the list's title)
  * - columns: { title: string, header: boolean, sortable: boolean }[] or string[]
  * - rows: {
- *      columns: (React.Node or string)[],
- *      props: { key: string, className: 'test', ...extraProps: object } - this property is mandatory and should contain a unique `key`, all additional properties are optional
+ *      columns: (React.Node or string or { title: string, key: string, ...extraProps: object}}[]
+                 Through extraProps the consumers can pass arbitary properties to the <td>
+ *      props: { key: string, ...extraProps: object }
```

Comment:
Was it intended to drop `className` from the documentation again? It was just introduced in the previous commit (pulled out of extraProps), and I don't see a corresponding code change here which would handle its removal?

Reply:
Hm, this was just cleaned up from the wrong commit, className is a normal property therefore encapsulated by extraProps, I though there is not need to document it seperately.

I can repost, to remove it in the commit that introduced this change 'className: 'test'' in the first place.